src/instances/RaidInstance.ts

```diff
@@ -1523,6 +1533,7 @@ export class RaidInstance {
                 .append(`the raid location. Your new location is: **${this._location}**.`)
                 .toString()
         });
+        this.sendLocToElite();
```

Comment:
This should be awaited, no? Or else you can throw a `.then()` after the call

Reply:
No need to await, nothing depends on this method completing.  can add a then().

Reply:
I see. Usually in said case I just stick a `.then()` so my IDE doesn't give me a warning. But not a big deal as you said